### ISSUE TITLE
Rework pong tutorial

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot-docs#149]
Some users reported on the Steam forum that the [Pong tutorial](http://docs.godotengine.org/en/latest/tutorials/step_by_step/simple_2d_game.html) is a bit hard to follow for newcomers: http://steamcommunity.com/app/404790/discussions/0/364040166684451403

Several issues:
- the code diverged a bit from the pong demo (likely the demo is better)
- there is no clear overview of how the things should be positioned or how the script should look like

**Comment by akien-mga** [comment]
Another user on IRC seemed to struggled with the old 1.0 screenshots. They should be remade for Godot 2.0.x.

**Comment by StraToN** [comment]
Reference to #128

**Comment by akien-mga** [comment]
Bump, another user reported that it was hard to follow: http://steamcommunity.com/app/404790/discussions/0/350542145688732254/

**Comment by reduz** [comment]
let's probably wait for 3.0 and then rework everything

**Comment by akien-mga** [comment]
Well the tutorial page itself could be reworked right now, it should not be too complicated, and it's definitely the page new users complain most about. It's a perfect job for @StraToN who has been quiet lately IMO... :grin:

**Comment by StraToN** [comment]
Alright, I'll work on it, starting tonight :)

**Comment by Appeltabak** [comment]
I was following the the tutorial and I also noticed a divergence: the picture of the pong nodes overview, so I started making a screen shot to fix this, but then I stumbled upon this issue 😁. @StraToN  I thought it may still be worth sharing this (as I didn't see it fixed yet):
![pong_nodes](https://cloud.githubusercontent.com/assets/6260588/18800192/fe10c8e2-81db-11e6-902b-5ea82af19a97.png)

**Comment by StraToN** [comment]
@Appeltabak Thanks. I'll use this image as a replacement for the current one.

**Comment by puppetmaster-** [comment]
I have create this "more godot like" pong game. Perhaps you can use it in the new tutorial.
https://github.com/puppetmaster-/godot-pong

**Comment by bojidar-bg** [comment]
@puppetmaster- Check this http://docs.godotengine.org/en/stable/tutorials/engine/project_organization.html for more "godot-like" organisation of the scenes. (though demos dump everything in one folder)

Also, it is `resources`, not `ressources` :smile:

**Comment by puppetmaster-** [comment]
@bojidar-bg thx for the info about organisation of scenes and the typo. I will reorganize it to have a even more "godot-like" 😄 .

**Comment by puppetmaster-** [comment]
It's now reorganized.

**Comment by vnen** [comment]
I think that a full Pong game using Godot-like structure is good to have as a demo. But the tutorial is aimed for total beginners, it might be too much. It would make sense only if divided into multiple parts, each to cover a different aspect of Godot.

**Comment by puppetmaster-** [comment]
I real like the idea to divided it  into multiple parts.

The  "Learning step by step" tutorial was my start in to godot. I went through it step by step.
But when I arrived on the simple 2d game part you got confused because you don't used what you have learned bevor.

**Comment by StraToN** [comment]
Did 9f9e183f7ee0f82094a21afa0ad08428f58b7a96 help for closing this issue?

**Comment by NathanLovato** [comment]
Chris produced a new getting started game, see https://github.com/godotengine/godot-docs/issues/444
